dlrover/python/elastic_agent/master_client.py

```diff
@@ -57,6 +57,28 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def report_event_to_master(
```

Comment:
No need to define function here. For details, please refer to the comments below.

Reply:
If so the implementation here should be unified: remove the ```self._master_client``` in ```ckpt_saver```.